### PR TITLE
Move contents from merkle subdir into top-level dir (resolves #45…

### LINKED ISSUE #45: Re-organize source tree, by removing the extra directory level

### PR DESCRIPTION
**Pull request by cryptonemo**
Minor updates.

### CODE REVIEW

**Comment by cryptonemo** [comment]
@dignifiedquire Some minor (naming) changes, and moves around the tree org as discussed in #45.  If this looks good to go, would you mind adding a version bump on top so that I can reference it nicely from the incoming rust-fil-proofs PR that uses this.

**Comment by cryptonemo** [comment]
Oh, I guess I could update the version here, but since you're the version master, figured it was proper to do it that way ;-)

**Comment by cryptonemo** [comment]
@dignifiedquire Updated, now ready for review!

**Comment by dignifiedquire** [comment]
sorry, I merged my other PR, this will need some rebasing to pull those changes in

File: src/test_xor128.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ use rayon::iter::ParallelIterator;
 use std::iter::FromIterator;
 use std::fmt;
 use std::hash::Hasher;
-use store::{Store, DiskStore, VecStore, LevelCacheStore};
```
Comment (dignifiedquire) [inline src/test_xor128.rs]: lets unexport the `LevelCacheStore` for now, or at least put a big warning on it to make clear that it does not work at the moment

File: src/store.rs [hunk 2]
```diff
@@ -434,6 +444,12 @@ impl<E: Element> Store<E> for DiskStore<E> {
         Ok(true)
     }
 
+    fn delete(config: StoreConfig) -> Result<bool> {
+        remove_file(StoreConfig::data_path(&config.path, &config.id))?;
+
+        Ok(true)
```
Comment (dignifiedquire) [inline src/store.rs]: what does the `bool` indicate, it seems to always return `true`

Comment (cryptonemo) [reply src/store.rs]: Yes, it does.  I'll update it to pass through the result of the remove_file call then `Result<()>`.

**Comment by cryptonemo** [comment]
@dignifiedquire Updated.

**Pull request merged**